ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/ngMaterial-theme.scss

```diff
@@ -58,6 +58,8 @@
       & .mat-mdc-dialog-content {
         @include utilities.box-reset;
 
+        overflow: visible;
```

Comment:
Don't know why this has been added, but it broke the UI. Will remove in next commit.